container/shared/src/main/resources/etc/container.init

```diff
@@ -215,6 +252,10 @@ case "$COMMAND" in
 		;;
 
 	status)
+		if ! check_upgrade_files; then
```

Comment:
Is it necessary to run this check when calling `status`?

Reply:
I suppose not... this is what the old behavior was, IIRC, but there's no reason it has to check there...

Reply:
Hm, actually I don't think that's what the old (opennms) behavior was, so never mind.  ;)